web/src/components/DateRangePicker.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { DateRangePicker, DateRangePickerProps } from "rsuite";
+import { Box } from "@mui/material";
+import "rsuite/dist/rsuite.min.css";
+
+export interface ICustomDateRangePickerProps extends DateRangePickerProps {
+  placeholder?: string;
+}
+
+const CustomDateRangePicker = ({
+  placeholder = "Select date range",
+  ...props
```

Comment:
para mas ma modify nato ang props
```suggestion
  ...props,
   format="yyyy/MM/dd"
```